dvc/path_info.py

```diff
@@ -341,5 +357,16 @@ class WebDAVURLInfo(URLInfo):
     @cached_property
     def url(self):
         return "{}://{}{}".format(
-            self.scheme.replace("webdav", "http"), self.netloc, self._spath
+            self.scheme.replace("webdav", "http"), self.netloc, self.spath
         )
+
+
+class GDriveURLInfo(CloudURLInfo):
```

Comment:
@julsal moving it here is - first, a cosmetic change, that I would not do as part of this PR, second, is arguable- e.g. my preference would be to move WebDav out of it- so that logic that is very specific to a particular cloud should stay in that cloud

Reply:
Sure, I'm reversing. The idea was to concentrate all related classes in the same file, but I agree that's a cosmetic change.